testapp/templates/testapp/config_doc.rst

```diff
@@ -0,0 +1,48 @@
+{% block link %}{{ link }}{% endblock %}
+
+{% block title %}{{ title }}{% endblock %}
+
+Settings Overview
+=================
+
+Enable/Disable configuration:
+"""""""""""""""""""""""""""""
+
+::
+
+    {% spaceless %}
+    {{ enable_settings }}
```

Comment:
Does template variable `enable_settings` work for you? 
I create `user.rst` documentation for testapp and this setting is rendered as empty

Reply:
No, it didn't work. I've deleted the testapp template and use the standard template for testing to avoid this kind of confusion.